pycfmodel/utils.py

```diff
@@ -1,20 +1,17 @@
 import re
-from typing import Any, Pattern
+from typing import Any, List, Pattern, Union
 
-from pycfmodel.constants import CONDITION_FUNCTIONS, IMPLEMENTED_FUNCTIONS
+from pycfmodel.constants import IMPLEMENTED_FUNCTIONS
 
 
 def is_resolvable_dict(value: Any) -> bool:
     return isinstance(value, dict) and len(value) == 1 and next(iter(value)) in IMPLEMENTED_FUNCTIONS
 
 
-def is_conditional_dict(value: Any) -> bool:
```

Comment:
Removing this is a major change. It could break libraries depending on this method